docs/core/plugins/sieve.md

```diff
@@ -383,153 +465,114 @@ when a plugin called `sieve_storage_ldap` is loaded.
+if a script is changed, then this entry must also be changed for it to be
+reloaded. Depending on which LDAP entry is configured, this can happen
+implicitly by the LDAP database itself (which is normally the case for the
+default `modifyTimestamp` entry).
 :::
 
-##### `user=<username>`
-
-Overrides the user name used for the lookup. Normally, the name of
-the user running the Sieve interpreter is used.
-
-##### LDAP Settings
-
-The configuration file is based on the [[link,auth_ldap]]. These
-parameters are specific to the Sieve ldap configuration:
-
-###### `sieve_ldap_filter`
-
-- Default: `(&(objectClass=posixAccount)(uid=%{user}))`
-- Values: [[link,settings_types_string]]
-
-The LDAP search filter that is used to find the entry containing the
-Sieve script.
-
-These variables can be used:
-
-| Variable | Long Name | Description |
-| -------- | --------- | ----------- |
-| `%{user}` | `%{user}` | username |
-| `%{user | username}` | `%{user | username}` | user part in user@domain, same as `%{user}` if there's no domain |
-| `%{user | domain}` | %{user | domain} | domain part in user@domain, empty if user there's no domain |
-| | `%{home}` | user's home directory |
-| | `%{name}` | name of the Sieve script |
-
-###### `sieve_ldap_script_attr`
-
-- Default: `mailSieveRuleSource`
-- Values: [[link,settings_types_string]]
+#### Example
 
-The name of the attribute containing the Sieve script.
-
-###### `sieve_ldap_mod_attr`
-
-- Default: `modifyTimestamp`
-- Values: [[link,settings_types_string]]
-
-The name of the attribute used to detect modifications to the LDAP entry.
+If support for the `ldap` script storage driver is compiled as a plugin, it
+needs to be added to the sieve_plugins setting before it can be used,
+e.g.:
 
-##### Example
+```
+sieve_plugins {
+  sieve_storage_ldap = yes
+}
+```
 
 ::: code-group
 ```[dovecot.conf]
-plugin {
-  sieve = ldap:/etc/dovecot/sieve-ldap.conf;bindir=~/.sieve-bin/
-}
-```
+sieve_script personal {
+  driver = ldap
+  bin_path = ~/.sieve-bin/
 
-```[/etc/dovecot/sieve-ldap.conf]
-# This file needs to be accessible by the Sieve interpreter running in LDA/LMTP.
-# This requires access by the mail user. Don't use privileged LDAP credentials
-# here as these may likely leak. Only search and read access is required.
+  # Don't use privileged LDAP credentials here as these may likely leak. Only
+  # search and read access is required.
 
-# Space separated list of LDAP hosts to use. host:port is allowed too.
-hosts = localhost
+  # Space separated list of LDAP hosts to use. host:port is allowed too.
+  ldap_hosts = localhost
 
-# Distinguished Name - the username used to login to the LDAP server.
-# Leave it commented out to bind anonymously.
-dn = cn=sieve,ou=Programs,dc=example,dc=org
+  # Distinguished Name - the username used to login to the LDAP server.
+  # Leave it commented out to bind anonymously.
+  ldap_dn = cn=sieve,ou=Programs,dc=example,dc=org
```

Comment:
Lets already rename these ldap settings to their final names. This is ldap_auth_dn.